src/ripple.js

```diff
@@ -78,8 +78,8 @@ var Ripple = {
             // No need to set positioning because ripple should be child of target and to it's relative position.
             // rippleContainer.style.left    = left + (((window.pageXOffset || document.scrollLeft) - (document.clientLeft || 0)) || 0) + "px";
             // rippleContainer.style.top     = top + (((window.pageYOffset || document.scrollTop) - (document.clientTop || 0)) || 0) + "px";
-            rippleContainer.style.width   = width + "px";
-            rippleContainer.style.height  = height + "px";
+            rippleContainer.style.width   = "100%";
+            rippleContainer.style.height  = height*1.2 + "px";
```

Comment:
why do we have to increase `height`? Isn't it safer to use set `height` in percent value of parent?

Reply:
BTW, because of this change `ripple` is going out of parent component boundary

Reply:
Ok, it's safer to don't touch this property, I will revert it.